pna_control/tone_finder.py

```diff
@@ -0,0 +1,191 @@
+import glob, csv
+import numpy as np
+from scipy.signal import find_peaks
+from matplotlib import pyplot as plt
+plt.rcParams['savefig.facecolor']='white'
+
+def loadVNA(FileList):
+    """
+    The data structure anticipated is single line header of columns with columns units in freq [Ghz], Power [dB], Phase [rad].
+    The data phase will be corrected without request if the phase exceeds 10 'degree' into radians.  
```

Comment:
Can you do the same if the frequency data is in Hz and not GHz? Our frequency data is usually in Hz and phase data is in degrees.